ArcGISRuntimeSDKQt_QMLSamples/Layers/BrowseOGCAPIFeatureService/BrowseOGCAPIFeatureService.qml

```diff
@@ -111,9 +112,7 @@ Rectangle {
         // Pop-up error message box
         MessageDialog {
             id: errorMessageBox
-            title: "Error message!"
             text: errorMessage
-            icon: StandardIcon.Warning
```

Comment:
Same here, I wonder if it is related to:

```
qrc:/Samples/Layers/BrowseOGCAPIFeatureService/BrowseOGCAPIFeatureService.qml:104: TypeError: Cannot read property 'loadStatus' of null
```

Reply:
Are you referring to `TypeError: Cannot read property 'loadStatus' of null`? That's a non-breaking error that just indicates that featureLayer was null when the button was trying to resolve whether or not it should be enabled. However featureLayer wasn't instantiated yet so it couldn't resolve featureLayer.loadStatus. I'll make the following change that enables the load button if the featureCollectionListComboBox.model has items. 

```js
-                    enabled: featureLayer.loadStatus !== Enums.LoadStatusLoading
+                    enabled: featureCollectionListComboBox.model.length > 0
```